Prox/Prox/Models/Place.swift

```diff
@@ -35,8 +35,8 @@ class Place {
     init?(fromFirebaseSnapshot data: FIRDataSnapshot) {
         guard data.exists(), data.hasChildren(),
                 let value = data.value as? NSDictionary,
-                let summary = value["description"] as? String ?? value["pullQuote"] as? String,
-                let name = value["id"] as? String, // TODO: change to name from id
+                let summary = value["summary"] as? String,
```

Comment:
I get no places when I run this code as there is no `summary` key, only a `description` key